src/arguments/channel.js

```diff
@@ -2,8 +2,12 @@ const { Argument } = require('klasa');
 
 module.exports = class extends Argument {
 
-	run(arg, possible, message) {
-		const channel = this.constructor.regex.channel.test(arg) ? this.client.channels.get(this.constructor.regex.channel.exec(arg)[1]) : null;
+	async run(arg, possible, message) {
+		// DM Channel support
+		const user = this.constructor.regex.userOrMember.test(arg) ? await this.client.users.fetch(this.constructor.regex.userOrMember.exec(arg)[1]) : null;
```

Comment:
What if you pass in an ID and this check throws? Say I pass in an ID for a channel, which matches the regex for userOrMember, users.fetch WILL throw

If anything this should be merged down into one ternary, if possible

Reply:
oh, the catch(() => null)s are gone, need to double check all args